src/utils/util.js

```diff
@@ -57,22 +64,24 @@ module.exports = class Util {
   }
 
   /**
-   * Get the lowest stateless rate of this shipment.
-   * @param {Rate[]} statelessRates
-   * @param {string[]} carriers
-   * @param {string[]} deliveryAccuracy
-   * @returns {Rate} Lowest rate
+   * Get the lowest rate from a provided list of stateless rates.
```

Comment:
```suggestion
   * Get the lowest rate from a provided list of rates.
```

Reply:
This is a function that works for normal rates as well as stateless rates, so purposely keeping the docstring generic.

Reply:
Yea, we need to remove the `stateless` in the docstring as this function became generic